### PR TITLE
Add_DisableCentralPackageVersions

### PR DESCRIPTION
**Pull request by cristinamanum**
Add DisableCentralPackageVersions for NuGet CPVM scenarios.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml [hunk 1]
```diff
@@ -185,4 +185,7 @@
                   ReadOnly="True"
                   Visible="False" />
 
+  <StringProperty Name="DisableCentralPackageVersions"
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml]: Nit: I would have called this "EnableCentralPackageVersions" and defaulted to true - negative options are hard to parse in the human brain.

Comment (davidwengier) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml]: Also nit: Once the name is settled, can this be moved to maintain alphabetical order of properties. Makes future diffing easier.

Comment (cristinamanum) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml]: Updated to address both of the comments.
